Source/KakapoDB.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  KakapoDB.swift
+//  KakapoExample
+//
+//  Created by Joan Romano on 31/03/16.
+//  Copyright © 2016 devlucky. All rights reserved.
+//
+
+import Foundation
+
+protocol KStorable {
+    var id: Int { get }
```

Comment:
id might be a string I guess and use `NSUUID`, no? otherwise we need a thread safe uuid (KakapoDB is not) or random one